src/ts/utils/decoration.ts

```diff
@@ -121,10 +107,12 @@ export const createDecorationsForMatch = (
       }`
     : DecorationClassMap[DECORATION_MATCH];
 
+
+  const matchColour = match.markAsCorrect ? "3ff200" :  match.category.colour;
```

Comment:
Nice-to-have: it'd be nice to apply this colour this with a class applied to the decoration, rather than hardcoding inline styles, as it would allow consumers to override the colour with CSS.

Reply:
Yeah I totally agree

Reply:
After discussing this further, @jonathonherbert and I feel this is acceptable as a temporary measure. The rationale for this is that categories owning custom colours is going to be replaced in favour of three standardised colours for Correct, Wrong and Possibly Wrong. When this happens we will adjust how the decoration styles are applied in favour of classes for each of these decorations. This will remove the inline styles, and at this point we can consider how a client may be able to customise these colours.